hackvote/app/controllers/events_controller.rb

```diff
@@ -0,0 +1,42 @@
+class EventsController < ApplicationController
+	include ApplicationHelper
+
+	def new
+		@event = Event.new
+		@cur_events = Event.where( completed: false)
+		@past_events = Event.where( completed: true)
+	end
+
+	def create
+		@event = Event.new(event_params)
+		if @event.save
+			cookies.permanent[@event.id] = 3
```

Comment:
also 3 should be a constant declared somewhere rather than a magic number defined here.